### DISCUSSION TITLE
v7 responsive example with changed columns

### DISCUSSION THREAD

**Discussion by dide0100** [TanStack/table#2549] [General]
Hi, first of all thank you for the great library. 
I would like to see an example of react table with different columns enabled based on screen size similar to the table library of https://www.npmjs.com/package/react-super-responsive-table

Basically my request is to change the table view to a more readable component in a hidden column maybe? if the screen resolution is is bellow a certain size (maybe with the help of media queries?)

CodeSandbox would be great.

Really appreciate the help.
Thank you

**Comment by dide0100** [comment]
not to much going on here @tannerlinsley would really appreciate help here

**Comment by gargroh** [comment]
React Table v7 is headless API i.e. you can control your component's dom, so feel free to try mentioned package with react table, it should work..

**Reply by dide0100** [reply]
Thank you for the answer @gargroh Im aware of the headless feature of the library, in fact im utilizing many of these feature but so far i have not been able to successfully get this to work. I have selective component rendering on other actions on the table but what im after is a good Codepen example of this like the ones from the library. I believe it could help a lot of people. Any help appreciated.

**Reply by gargroh** [reply]
Currently no example like that exists, feel free to open a code sandbox for the same, community will be happy to contribute in it...
